torch/nn/quantized/modules/activation.py

```diff
@@ -149,3 +149,24 @@ def _get_name(self):
     def from_float(cls, mod):
         scale, zero_point = mod.activation_post_process.calculate_qparams()
         return cls(float(scale), int(zero_point), mod.negative_slope, mod.inplace)
+
+class Sigmoid(torch.nn.Sigmoid):
+    r"""This is the quantized equivalent of :class:`~torch.nn.LeakyReLU`.
```

Comment:
The docs are off here, can we refer to sigmoid instead of leakyRelu?

Reply:
oh sorry, will fix in next PR